sdk/python/arvados/arvfile.py

```diff
@@ -516,7 +517,7 @@ def _block_prefetch_worker(self):
                     return
                 self._keep.get(b)
             except Exception:
-                pass
+                _logger.error(traceback.format_exc())
```

Comment:
I believe the purpose of this Pokémon exception handler is to suppress failures from unknown rare errors. Therefore, the message should scarcely be seen by users of the SDK. This being the case, it seems reasonable for it to be intended for developers.

With regards to the extra import: the import statement could be moved in the `except` block?